PluralKit.Bot/Commands/SystemEdit.cs

```diff
@@ -91,6 +92,47 @@ await ctx.Reply(embed: new EmbedBuilder()
                 await ctx.Reply($"{Emojis.Success} System description changed.");
             }
         }
+
+        public async Task Color(Context ctx) {
+            ctx.CheckSystem();
+
+            if (await ctx.MatchClear())
+            {
+                var patch = new SystemPatch {Color = Partial<string>.Null()};
+                await _db.Execute(conn => _repo.UpdateSystem(conn, ctx.System.Id, patch));
+
+                await ctx.Reply($"{Emojis.Success} System color cleared.");
+            }
+            else if (!ctx.HasNext()) 
+            {
+                if (ctx.System.Color == null) 
```

Comment:
Does/should this allow you to look up other systems' colors?

Reply:
There's not a command for specifically looking up a system's color by specifying a target. But pulling up the system card in general shows the color and hex code field like it would on the member card.